test/elixir/test/view_pagination_test.exs

```diff
@@ -59,7 +59,7 @@ defmodule ViewPaginationTest do
 
     query_function = "function(doc) { emit(doc.integer, null); }"
 
-    100..0//-1
+    100..0//1
```

Comment:
I think it's for string slicing only?

Otherwise for range we'd end up with an empty range:

```
> Enum.to_list(100..0//-1)
[100, 99, 98, 97, 96, 95, 94, 93, 92, 91, 90, 89, 88, 87, 86, 85, 84, 83, 82,
 81, 80, 79, 78, 77, 76, 75, 74, 73, 72, 71, 70, 69, 68, 67, 66, 65, 64, 63, 62,
 61, 60, 59, 58, 57, 56, 55, 54, 53, 52, 51, ...]

> Enum.to_list(100..0//1)
[]
```